services/rollout-service/pkg/cmd/server.go

```diff
@@ -157,6 +159,33 @@ func runServer(ctx context.Context, config Config) error {
 	broadcast := service.New()
 	shutdownCh := make(chan struct{})
 	ready := false
+	versionC := versions.New(overview)
+
+	backgroundTasks := []setup.BackgroundTaskConfig{
+		{
+			Name: "consume argocd events",
+			Run: func(ctx context.Context) error {
+				return service.ConsumeEvents(ctx, appClient, versionC, broadcast, func() { ready = true })
+			},
+		},
+		{
+			Name: "consume kuberpult events",
+			Run: func(ctx context.Context) error {
+				return versionC.Subscribe(ctx, broadcast)
```

Comment:
can we rename `Subscribe` to `ConsumeEvents` or vice versa?

Reply:
Done